nix-tools-regenerate.nix

```diff
@@ -1,11 +1,10 @@
 # A script for generating the nix haskell package set based on stackage,
 # using the common convention for repo layout.
 
-{ lib, stdenv, path, writeScript, nix-tools, coreutils, findutils, gawk
-, nix, nix-prefetch-scripts }:
+{ lib, stdenv, path, writeScript, nix-tools, coreutils, findutils }:
 
 let
-  deps = [ nix-tools coreutils gawk nix nix-prefetch-scripts findutils ];
```

Comment:
I'm not sure we can drop `nix-prefetch-scripts`. I believe `stack-to-nix` will call out to the prefetch commands for source repositories. See

https://github.com/input-output-hk/nix-tools/blob/bb5700d04f0ff2d1a3a7b3278121c1ec67898793/lib/Distribution/Nixpkgs/Fetch.hs#L145

Reply:
Yes, we can drop it because the nix-tools programs are [wrapped](https://github.com/input-output-hk/haskell.nix/blob/4c5cf05ab4e433cb6de1adc56f16166542d55a02/nix-tools/default.nix#L50-L54).

Reply:
I see. I've added input-output-hk/haskell.nix#208